projects/packages/mcp/src/class-mcp.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Jetpack MCP Package
+ *
+ * @package automattic/jetpack-mcp
+ */
+
+namespace Automattic\Jetpack;
+
+use Automattic\Jetpack\AbilitiesRegistry\Registry\AbilityRegistry;
+
+/**
+ * Jetpack MCP Package class.
+ */
+class Mcp {
+
+	const PACKAGE_VERSION = '0.1.0-alpha';
+
+	/**
+	 * Initialize the MCP package.
+	 */
+	public static function init(): void {
+		// Prevent multiple executions.
+		static $loaded = false;
+		if ( $loaded ) {
+			return;
+		}
+		$loaded = true;
+
+		// Only load for Automatticians on WordPress.com.
+		// @phan-suppress-next-line PhanUndeclaredFunction
+		if ( ! function_exists( 'is_automattician' ) || ! is_automattician() ) {
```

Comment:
We could handle this with a feature flag by using a filter

Reply:
I'm removing this. I don't see why we need to feature flag the Ability Registry.